### PR TITLE
Fix metric meter initialization

### PR DESCRIPTION
**Pull request by Sushisource**
Prometheus initialization would explode because it needs tokio to exist, but was being initialized before tokio was.

### CODE REVIEW

File: packages/test/src/test-prometheus.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+if (RUN_INTEGRATION_TESTS) {
+  test.serial('Prometheus metrics work', async (t) => {
+    const logger = new DefaultLogger('DEBUG');
+    Runtime.install({
+      logger,
+      telemetryOptions: {
+        metrics: {
+          prometheus: {
+            bindAddress: '0.0.0.0:9090',
```
Comment (mjameswh) [inline packages/test/src/test-prometheus.ts:18]: Can we use a random port for this, and bind to 127.0.0.1 instead?

```
async function getPortFree() {
    return new Promise( res => {
        const srv = net.createServer();
        srv.listen(0, () => {
            const port = srv.address().port
            srv.close((err) => res(port))
        });
    })
}
```

**Comment by mjameswh** [comment]
Not elegant, but that should do the work until we revisit `CoreRuntime::new`. Thanks!

**Review by mjameswh — APPROVED** [review]
Please make the prom test not bind on `0.0.0.0:9090`. Thanks!

File: packages/test/src/test-prometheus.ts (hunk 1 above)
Comment (Sushisource) [reply packages/test/src/test-prometheus.ts:18]: I knew I'd need to change that and forgot. Thanks for the reminder.

**Pull request merged**